plugins/lookup/etcd.py

```diff
@@ -54,13 +54,13 @@
 
 EXAMPLES = '''
     - name: "a value from a locally running etcd"
-      debug: msg={{ lookup('etcd', 'foo/bar') }}
+      ansible.builtin.debug: msg={{ lookup('etcd', 'foo/bar') }}
```

Comment:
For the plugins, we have to make sure that the plugin name is in FQCN in the examples. (Not in this PR though :) )